x-pack/solutions/observability/plugins/slo/server/saved_objects/slo.ts

```diff
@@ -49,13 +49,41 @@ export const slo: SavedObjectsType = {
         },
       ],
     },
+
+    2: {
+      changes: [
+        {
+          type: 'mappings_addition',
+          addedMappings: {
+            artifacts: {
+              properties: {
+                dashboards: {
+                  properties: {
+                    refId: { type: 'keyword' },
+                  },
+                },
+              },
+            },
+          },
+        },
+      ],
+    },
   },
   mappings: {
     dynamic: false,
     properties: {
       id: { type: 'keyword' },
       name: { type: 'text' },
       description: { type: 'text' },
+      artifacts: {
+        properties: {
+          dashboards: {
+            properties: {
+              refId: { type: 'keyword' },
```

Comment:
If we were using the dashboard id, we could find every SLO with a specific dashboard id. I guess that's possible with the references also, can you confirm?

Reply:
We can leverage the `hasReference` property of the find method in the saved objects client.